givitsite/coordinate/admin.py

```diff
@@ -1,3 +1,6 @@
+import sys
 from django.contrib import admin
+from . models import CoordinatedItems
 
-# Register your models here.
+sys.path.append('../')
```

Comment:
What are you trying to accomplish with 
sys.path.append('../')
?

Reply:
I want  to import the ItemsFound, ItemRequest DB models from a sibling directory - this is problematic as in our architecture the coordinator directory is a root directory and 'from ..friendreq/models import ItemsFound, ItemRequest' does not work, adding (..) to sys.path allows simmleslly importing each model from a sibling directory.

Reply:
So you're using this path change in other modules.
I suggest adding a comment indicating this, as it isn't obvious what the usage is in this file.
Another place to add this might be in the __init__.py as it is for the entire system and not this specific file

Reply:
Done